di/src/test/java/com/example/di/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.di
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
add an empty line at the end of the file pls

Reply:
done!